docs/sources/fundamentals/intro-to-prometheus/index.md

```diff
@@ -0,0 +1,110 @@
+---
+aliases:
+  - ../basics/timeseries/
+description: Introduction to Prometheus
+keywords:
+  - grafana
+  - intro
+  - Prometheus
+  - metrics
+  - time series
+title: What is Prometheus?
+weight: 300
+---
+
+# What is Prometheus?
+
+Observability focuses on understanding the internal state of systems based on the data they produce, which helps determine if the infrastructure is operating as expected. Prometheus is a core technology for system observability. However, Prometheus can be a confusing term because it is used in different contexts. Understanding Prometheus basics, why it’s valuable, and how people use it in practice will help you use Grafana.
```

Comment:
It wasn't contextually relevant in this paragraph, and as other comments allude to, getting into what metrics are creates the need to explain and outline the entire "taxonomy":  Telemetry = logs + metrics + traces.  There are additional complicated bits that we'd do well to avoid in this intro article, like how prometheus can do more than metrics.  That's a much more advanced topic which I think would confuse the discussion here.  How to get around all of this complexity without wrestling with it directly -- right up front in the first article -- is a real challenge.

It's like claiming that Newtonian physics is correct.  It *is*, but it isn't the whole story.  But if you're trying to teach newtonian physics, you really don't want to get into a discussion of quantum mechanics.

Reply:
Leaving that statement where it is.